frontend/src/lib/k8s/cluster.ts

```diff
@@ -55,6 +56,16 @@ export interface ApiListOptions extends QueryParameters {
   namespace?: string | string[];
 }
 
+export interface KubeManagedFields {
+  apiVersion: string;
+  fieldsType: string;
+  fieldsV1: { map: Map<string, string[]> };
```

Comment:
Is it going to be a map, or `{[key: string]: string[]}`.
I am asking because I don't think we have maps anywhere else in the types.

Reply:
Changing this to `{[key:string]:string[]}` as we are not dependent on the `Map` functionalities in this case.